src/utils.ts

```diff
@@ -1,22 +1,22 @@
-const KNOWN_CHAINS = new Map([
-  ['1', 'Mainnet'],
-  ['3', 'Ropsten'],
-  ['4', 'Rinkeby'],
-  ['5', 'Goerli'],
-  ['100', 'xDai'],
+import { Account, EthereumProvider } from './types'
+
+const KNOWN_CHAINS = new Map<number, string>([
+  [1, 'Mainnet'],
```

Comment:
I wonder if we would be interested in using the names in https://chainid.network/ instead, or if it'd be too verbose?

Do we need to use the names in this library (I would treat this somewhat as a user concern, since it's up to them how they'd like to define the UI per network)

Reply:
> Do we need to use the names in this library (I would treat this somewhat as a user concern, since it's up to them how they'd like to define the UI per network)

Yes I agree, let’s remove them and `networkName` :+1:  (I’ll do this in a separate PR).